worker/process-image.js

```diff
@@ -103,6 +109,41 @@ function uploadToS3 (s3, path, key, metadata, thumbPath, callback) {
   });
 }
 
+function translateJPEG2000 (ext, path, key, callback) {
+  if (ext !== '.jp2') {
+    return callback(null, path, key);
+  }
+
+  if (!config.gdalTranslateBin) {
+    throw new Error('GDAL bin path missing.');
+  }
+
+  log(['debug'], 'Converting JPEG2000 to TIFF.');
+  var parsedPath = pathTools.parse(path);
+  var outPath = pathTools.join(parsedPath.dir, parsedPath.name) + '.tif';
+
+  var args = [
+    '-of', 'GTiff',
+    path, outPath,
+    '-co', 'TILED=yes',
+    '-co', 'COMPRESS=DEFLATE',
+    '-co', 'PREDICTOR=2',
+    '-co', 'SPARSE_OK=yes',
+    '-co', 'BLOCKXSIZE=256',
+    '-co', 'BLOCKYSIZE=256',
```

Comment:
@mojodna I had to remove the `INTERLEAVE=BAND` flag from this command because the utility for generating the thumbnails, sharp, wasn't able to process larger band-interleaved images.